game/requesthandler.cc

```diff
@@ -5,15 +5,15 @@
 #include <cstdint>
 
 #ifdef USE_WEBSERVER
-RequestHandler::RequestHandler(Songs& songs):m_songs(songs)
+RequestHandler::RequestHandler(Game &game, Songs& songs): m_game(game), m_songs(songs)
 {
 }
-RequestHandler::RequestHandler(std::string url, Songs& songs):m_listener(url),m_songs(songs)
+RequestHandler::RequestHandler(Game &game, std::string url, Songs& songs): m_game(game), m_listener(url),m_songs(songs)
 {
-    m_listener.support(web::http::methods::GET, std::bind(&RequestHandler::Get, this, std::placeholders::_1));
```

Comment:
Please revert whitespace changes in `webserver` and `requesthandler` files, Whichever webserver PR we end up going with, this is going to cause a merging nightmare.

Reply:
All whitespace-only changes reverted for this file.